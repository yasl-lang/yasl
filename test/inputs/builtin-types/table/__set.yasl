let x = { 3:'three', 1:'one', 2:'two'}
x[1] = 'un'
echo x
