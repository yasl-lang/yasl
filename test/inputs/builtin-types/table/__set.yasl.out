{2: two, 1: un, 3: three}
